cognite_toolkit/_cdf_tk/loaders/_base_loaders.py

```diff
@@ -56,17 +57,20 @@ class Loader(ABC):
     _doc_base_url: str = "https://api-docs.cognite.com/20230101/tag/"
     _doc_url: str = ""
 
-    def __init__(self, client: ToolkitClient, build_dir: Path | None) -> None:
+    def __init__(self, client: ToolkitClient, build_dir: Path | None, console: Console | None = None) -> None:
```

Comment:
This is added such that when using the status from rich, we need to specify the console to get warnings to print out appropriately.